db/migrate/20190716115336_create_posts.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[5.2]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+
+      t.timestamps
```

Comment:
I see no connection to the user here....

Reply:
Thanks, so we mixed up the user and visitor. The role we'd like to have in this feature is visitor, since its most related to the MVP. We'll go on and make changes for comments below so that this feature is all about the visitor.

Reply:
What is the difference between `User` and `>visitor`?

Reply:
a visitor doesn't have to be logged in in order to use certain features, but a visitor becomes a user when logged in

Reply:
Okay, so what is the reason for a visitor to log in?

Reply:
Okay so yes we will need to change it again, this time to make use of the users that the rest of our team already created. We agree it doesn't really make sense to sign up to the community if features are accessible already as a visitor...